examples/plot_replay_experiment.py

```diff
@@ -0,0 +1,149 @@
+"""
+Replay and experiment
+=====================
+
+Active Learning experiments can be long and costly. For this reason,
+it is useful to be able to resume an experiment if an error happened.
+To achieve that, cardinal allows to store intermediate variables,
+such as selected samples, in a cache. Users can therefore resume
+an interrupted experiment, using the ResumeCache, or replay an entire
+experiment to perform additional computation such as metric using the
+ReplayCache.
+"""
+
+import shutil
+import os
+import numpy as np
+import dataset
```

Comment:
just a placeholder as per our discussion (probably it's easier to add this as -optional?- dependence). 
Such a poor naming though :)

Reply:
ping :)

Reply:
Yay!